docs/user/viewer-preferences.md

```diff
@@ -1,15 +1,16 @@
 # Adding Viewer Preferences
 
-It is possible to set viewer preferences of a PDF file.
-These properties are described in Section 12.2 of the [PDF 1.7 specification](https://opensource.adobe.com/dc-acrobat-sdk-docs/pdfstandards/PDF32000_2008.pdf).
+It is possible to set viewer preferences of a PDF file.  These properties are
```

Comment:
Do we really have any benefits from this? Additionally, there is a whitespace too much.

Reply:
Will change this.